src/home/TabHome.tsx

```diff
@@ -134,9 +134,29 @@ function TabHome(_props: Props) {
       })
   }
 
+  const cashOutTokens = useCashOutTokens()
+
   function onPressWithdraw() {
+    const numAvailableCashOutTokens = cashOutTokens.filter(
+      (token) => !token.balance.isZero()
+    ).length
+    if (
+      numAvailableCashOutTokens === 1 ||
```

Comment:
Navigate directly to enter amount if:
- user only has 1 token with balance OR
- there is only 1 available cash out token
otherwise show bottom sheet